### ISSUE TITLE
The Creator column can only be filtered by username

### ISSUE THREAD

**Issue by ane-gabriela** [xwikisas/application-mocca-calendar#107]
Preconditions: Have at least 3-4 calendars created. with different users that have/don't have First and Last name.

1. Click on Calendar from Applications
1. Scroll down to the "Overview of all calendars" section
1. Filter the calendars by Creator
![Creator](https://github.com/xwikisas/application-mocca-calendar/assets/40691112/9f233fbf-2c65-4313-8e2d-a86e6ca2f750)

Expected results: The filter works properly.

Actual results: The filter works only when using the username. The filter doesn't work by First or Last name.

![User](https://github.com/xwikisas/application-mocca-calendar/assets/40691112/5bd9df3f-7a3b-4fb2-8d25-10cbb717f121)

![FirstName](https://github.com/xwikisas/application-mocca-calendar/assets/40691112/7e6626d7-2d07-4faf-9e2a-bc5a4a85ee90)

Environment: Windows 11, XWiki 14.10.10 with MySQL 8.0, Chrome 114, Calendar Application (Pro) 2.13

**Comment by lucaa** [comment]
@ane-gabriela do you have any information about whether this is a regression  (for XWiki 14.10.10 or Calendar app 2.1.3 ) or it has been like this for a while already?

**Comment by ane-gabriela** [comment]
It appears to be an old issue - Tested with Calendar 2.12 and XWiki 14.10.5 and the issue reproduces

![14 10 5](https://github.com/xwikisas/application-mocca-calendar/assets/40691112/a7371266-c377-4f97-b5c8-8c78ec4dc7e5)

**Comment by snazare** [comment]
The issue is caused by the livetable feature, not the extension. The livetable is sorting based on the database property, not the display name.

**Comment by lucaa** [comment]
> The issue is caused by the livetable feature, not the extension. The livetable is sorting based on the database property, not the display name.

Indeed, the sorting is based on the username but for filtering, in the standard livetable there is a picker to pick a user based on first name + last name and then filter by this user.
